src/stores/UserStore.js

```diff
@@ -69,6 +71,8 @@ export default class UserStore {
   }
 
   getPercentageCompletion(collectionname){
+    // This could be made prettier with a switch. Even better (the only 2 lines)
+    // use a data object for the percentage divider) - Harold
```

Comment:
I'm not a big fan of switch statements. My refactoring book considers it a "bad smell" and gives advice how to refactor. But it's still cleaner than tons of if's.